controllers/taloscontrolplane_controller.go

```diff
@@ -307,7 +307,9 @@ func (r *TalosControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			if err := r.bootstrapCluster(ctx, cluster, ownedMachines); err != nil {
 				conditions.MarkFalse(tcp, controlplanev1.MachinesBootstrapped, controlplanev1.WaitingForTalosBootReason, clusterv1.ConditionSeverityInfo, err.Error())
 
-				return ctrl.Result{}, err
+				logger.Info("Bootstrap failed, retrying in 5 seconds", "error", err)
```

Comment:
nit: 

```suggestion
				logger.Info("bootstrap failed, retrying in 5 seconds", "error", err)
```

Reply:
Other logs start with the capital letter here :( Maybe we should change them everywhere to start with the lowercase letter

Reply:
ah, ok then... sorry for the noise